courses/free-courses-pt_BR.md

```diff
@@ -132,6 +132,7 @@
 
 * [Curso de Git](https://www.youtube.com/playlist?list=PLucm8g_ezqNq0dOgug6paAkH0AQSJPlIe) - Bóson Treinamentos
 * [Curso de Git e GitHub: grátis, prático e sem usar comandos no terminal](https://www.youtube.com/playlist?list=PLHz_AreHm4dm7ZULPAmadvNhH6vk9oNZA) - Gustavo Guanabara
+* [Curso Git e Github 2024](https://www.youtube.com/watch?v=M3aHjXSqFtE&list=PLHbGjxRVA_FmFRF1-OqWaWwqhIiS9Cg0w) - Carlos Uchoa
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes